python/kfserving/docs/KFServingClient.md

```diff
@@ -20,6 +20,8 @@ KFServingClient | [set_credentials](#set_credentials) | Set Credentials|
 KFServingClient | [create](#create) | Create KFService|
 KFServingClient | [get](#get)    | Get or watch the specified KFService or all KFServices in the namespace |
 KFServingClient | [patch](#patch)  | Patch the specified KFService|
+KFServingClient | [replace](#replace) | Replace the specified KFService|
```

Comment:
what does it mean to replace? do you mean the default version will be replaced? what if it has a canary will replace fail...
maybe better to say "Replace the default version of the specified KFService"

Reply:
actually I dont understand what this does - can please help me understand?

Reply:
@rakelkar Please see the PR describe for the reason to introduce `replace` API. I had a long discussion with @yuzisun offline and summarized there.